libs/librrgraph/src/base/rr_graph_storage.h

```diff
@@ -510,6 +550,8 @@ class t_rr_graph_storage {
         node_storage_.resize(size);
         node_ptc_.resize(size);
         node_layer_.resize(size);
+        node_bend_start_.resize(size);
+        node_bend_end_.resize(size);
```

Comment:
Shouldnt be resizing bend_start and bend_end when they aren't needed (only needed for tileable rr-graph)